packages/utils/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './src/constants.js'
```

Comment:
Let's scope everything like we do in ui and core, i.e. end result when using utils package should be something like

```ts
import { TypeUtil, PresetUtil, ConstUtil } from "@web3modal/utils"
```